lib/mousestats/index.js

```diff
@@ -31,7 +31,7 @@ var MouseStats = module.exports = integration('MouseStats')
 MouseStats.prototype.initialize = function(page){
   var number = this.options.accountNumber;
   var path = number.slice(0,1) + '/' + number.slice(1,2) + '/' + number;
-  var cache = Math.floor(new Date().getTime() / 60000);
+  var cache = Math.floor(new Date().getTime() / 180000);
```

Comment:
Just for caching purposes. However, you are right. 30 minutes would be too much. It was a mistake in conversion. It should remained unchanged (1*60*1000).